### PR TITLE
Bugfixes - Image with Hover Cards (Five Institutions)

### PR DESCRIPTION
**Pull request by mdickson-adbe**
Fix various visual issues with spacing, button colors.

Fix # https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-266, https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-271

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/en/who-we-are
- After: https://bugfixes-5insts--world-bank--aemsites.hlx.live/en/who-we-are

### CODE REVIEW

**Review by mkbansal1 — CHANGES_REQUESTED** [review]
@mdickson-adbe  I see double borders on "Learn more" link
image

**Comment by thomsebastin** [comment]
@mdickson-adbe I still see the double border. Could you please have a look at this?

Screenshot 2024-09-19 at 7 35 29 PM

**Comment by mdickson-adbe** [comment]
> @mdickson-adbe I see double borders on "Learn more" link image

Please re-check. I could have sworn I removed that border-bottom but I guess I hadn't.

File: blocks/organizational-cards/organizational-cards.css [hunk 1]
```diff
@@ -323,6 +314,20 @@ main .section > div.organizational-cards-wrapper {
...
   }
 
+  .organizational-card.collapsed .card-content-wrapper {
+    padding-bottom: unset;
+    border-bottom: unset;
+  }
+
+  .organizational-card .card-content-wrapper .collapse-btn {
+    background: var(--primary-blue-50);
+    border: 1px solid var(--primary-blue-50);
```
Comment (thomsebastin) [inline blocks/organizational-cards/organizational-cards.css:324]: all variables that starts with `primary` are by us and we will get rid of them soon. I am approving this PR but kindly follow up with appropriate new vars in new PR.

**Review by thomsebastin — APPROVED** [review]

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**